build.gradle

```diff
@@ -66,5 +66,8 @@ static def getTag() {
         else
             tagVersion = tagVersionToken[0]
     }
+    if (tagVersion.length() > 1 && tagVersion.startsWith('v')){
```

Comment:
Isn't `tagVersion.length() > 1 &&` obsolete ?

Reply:
GitHub still recommend to add this prefix and this code prevents to add this prefix into java version.

Reply:
Why ? What is the purpose of it ? Maybe there is a valid reason, but I can't image why (currently)

When there is a `v`, is there `w`, 'x'.... as well ?